x-pack/metricbeat/module/oracle/tablespace/metricset_test.go

```diff
@@ -0,0 +1,45 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+// +build integration
+
+package tablespace
+
+import (
+	"testing"
+
+	"github.com/elastic/beats/x-pack/metricbeat/module/oracle"
+
+	"github.com/stretchr/testify/assert"
+
+	_ "gopkg.in/goracle.v2"
+
+	mbtest "github.com/elastic/beats/metricbeat/mb/testing"
+)
+
+func TestData(t *testing.T) {
+	t.Skip("Skip until a proper Docker image is setup for Metricbeat")
+
+	f := mbtest.NewReportingMetricSetV2Error(t, getConfig())
+
+	events, errs := mbtest.ReportingFetchV2Error(f)
+	if len(errs) > 0 {
+		t.Fatalf("Expected 0 error, had %d. %v\n", len(errs), errs)
+	}
+	assert.NotEmpty(t, events)
```

Comment:
These lines are not needed, all this is already checked in the writer after checking if the test need to be skipped.

Reply:
Good catch!